### PR TITLE
Email campaign dynamic section

### PR DESCRIPTION
**Pull request by ilappe**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34188

## Description
Add the new incident fields and scripts for perform actions on campaign incidents

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            readable_output = get_incidents_info_md(incidents)
+        else:
+            readable_output = NO_CAMPAIGN_INCIDENTS_MSG
+
+        result = CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='',
+            outputs_key_field=''
+        )
+        return_results(result)
```
Comment (teizenman) [inline Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py:94]:
```suggestion
        return_results(readable_output)
```

I recommend going to the implementation of `return_results` to understand it better.

File: Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+def main():
+    try:
+        action = get_custom_field(ACTION_ON_CAMPAIGN_FIELD_NAME).lower()
+        ids = get_custom_field(SELECT_CAMPAIGN_INCIDENTS_FIELD_NAME)
+        if ALL_OPTION in ids:
+            ids = get_campaign_incident_ids()
+
+        res = ACTIONS_MAPPER[action](ids, action)
+        demisto.results(res)
```
Comment (teizenman) [inline Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py:85]:
```suggestion
        return_results(res)
```

File: Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+ALL_OPTION = 'All'
+COMMAND_ERROR_MSG = 'Error occurred while trying to perform \"{action}\" on the selected incident ids: {ids}'
+ACTION_ON_CAMPAIGN_FIELD_NAME = 'actionsoncampaignincidents'
+SELECT_CAMPAIGN_INCIDENTS_FIELD_NAME = 'selectcampaignincidents'
+
+COMMAND_SUCCESS = 'The following incidents was successfully {action}, {ids}'
```
Comment (teizenman) [inline Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py:9]:
```suggestion
COMMAND_SUCCESS = 'The following incidents was successfully {action}: {ids}'
```

**Review by teizenman — CHANGES_REQUESTED** [review]
1. Add docstring everywhere.
2. Notice that some of the comments are relevant to more than one place, so please review each comment and look for more places to fix it.

File: Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py (hunk 1 above)
Comment (ilappe) [reply Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py]: We should use CommandResult as the output should displayed as MD in the dynamic section

Comment (teizenman) [reply Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py]: It doesn't work without the `CommandResults` object?

Comment (ilappe) [reply Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py]: Without `CommandResults` it's appear in the section as simple text and not convert to a table

**Review by teizenman — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@teizenman Done.

**Pull request merged**